src/oatpp/core/async/worker/IOEventWorker_common.cpp

```diff
@@ -40,8 +40,8 @@ IOEventWorker::IOEventWorker(IOEventWorkerForeman* foreman, Action::IOEventType
   , m_foreman(foreman)
   , m_specialization(specialization)
   , m_running(true)
-  , m_eventQueueHandle(-1)
-  , m_wakeupTrigger(-1)
+  , m_eventQueueHandle(v_io_handle (-1))
```

Comment:
It might be a good idea to declare and `INVALID_IO_HANDLE` constant [here](https://github.com/oatpp/oatpp/blob/master/src/oatpp/core/IODefinitions.hpp#L36) together with `v_io_handle`.

Which is `-1` on Linux and `INVALID_SOCKET` on Windows.

Reply:
Is it fine to `#include <Windows.h>` at that point in order to get `INVALID_SOCKET` constant?

Reply:
Yes, you can include windows.h in `IODefinitions.hpp` file the v_io_handle type is defined there, so it's a good idea to define the corresponding `INVALID_IO_HANDLE` constant in the same place. Then we can use the `INVALID_IO_HANDLE` in the rest of the code.